### ISSUE TITLE
`TypeError: prettier.resolveConfig.sync is not a function`

### ISSUE THREAD

**Issue by hamirmahal** [microsoft/vscode-eslint#2000]
I see `TypeError: prettier.resolveConfig.sync is not a function` in the `OUTPUT` tab for `ESLint` in VSCode.

**Comment by hamirmahal** [comment]
Here's the full error message.

```
[Error - 11:39:52 AM] An unexpected error occurred:
[Error - 11:39:52 AM] TypeError: prettier.resolveConfig.sync is not a function
Occurred while linting /home/hamir/ui-shared/packages/component-library/src/components/Footer/FooterView.tsx:1
Rule: "prettier/prettier"
    at Program (/home/hamir/ui-shared/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js:138:40)
    at ruleErrorHandler (/home/hamir/ui-shared/node_modules/eslint/lib/linter/linter.js:1076:28)
    at /home/hamir/ui-shared/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/home/hamir/ui-shared/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/home/hamir/ui-shared/node_modules/eslint/lib/linter/node-event-generator.js:297:26)
    at NodeEventGenerator.applySelectors (/home/hamir/ui-shared/node_modules/eslint/lib/linter/node-event-generator.js:326:22)
    at NodeEventGenerator.enterNode (/home/hamir/ui-shared/node_modules/eslint/lib/linter/node-event-generator.js:340:14)
    at CodePathAnalyzer.enterNode (/home/hamir/ui-shared/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:803:23)
    at /home/hamir/ui-shared/node_modules/eslint/lib/linter/linter.js:1111:32
```

**Comment by hamirmahal** [comment]
![Image](https://github.com/user-attachments/assets/2c9ef8a2-0d16-492e-baf5-0c5d8e74b089)

**Comment by dbaeumer** [comment]
This issue has been closed because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by hamirmahal** [comment]
I get a similar error in the terminal, but this is in a closed-source workspaces project so I don't have a readily available repository that reproduces the issue at this point in time.

**Comment by dbaeumer** [comment]
If it reports in the terminal as well this is very likely a problem with ESLint itself and not with the extension.
